### DISCUSSION TITLE
Installation on PC?

### DISCUSSION THREAD

**Discussion by dagadeta** [Screenly/Anthias#2129] [Q&A] [answered]
In the [README Compatibility section](https://github.com/Screenly/Anthias#white_check_mark-compatibility), it says that Anthias is compatible with PCs (x86 64-bit). However, I can not find a build for PCs in the assets of the [current release (v0.19.1)](https://github.com/Screenly/Anthias/releases/tag/v0.19.1).

How can I install Anthias on a PC and is it its own OS (like for the Pi) or an application then? And if the last option is the case, with which Operating systems is it compatible?

**Comment by nicomiguelino** [comment, marked as answer]
For x86 devices, you have to install Debian 12 first, then run `bash <(curl -sL https://install-anthias.srly.io)`, as seen in [this documentation](https://github.com/Screenly/Anthias/blob/master/docs/installation-options.md#installing-on-raspberry-pi-os-lite-or-debian).

As of the moment, we don't have any pre-built images for x86 yet.

**Comment by vpetersson** [comment]
Installing on x86 PCs is supported: install Debian, then run bash <(curl -sL https://install-anthias.srly.io). This thread already has an accepted answer, so closing as resolved.
